packages/language-server/src/ls-config.ts

```diff
@@ -478,16 +478,25 @@ export class LSConfigManager {
         };
     }
 
-    getTsUserPreferences(lang: TsUserConfigLang, workspacePath: string | null): ts.UserPreferences {
+    getTsUserPreferences(
+        lang: TsUserConfigLang,
+        normalizedWorkspacePath: string | null
```

Comment:
you changed the name here - just for readability, or is there a corresponding normalization missing in another file?